flimfit.html

```diff
@@ -0,0 +1,878 @@
+---
+layout: default
+title: Using FLIMfit with OMERO
+menu-id: l1-other
+pdfs:
+    - flimfit.pdf
+description: FLIMfit is a software tool from, Imperial College London, that is designed to facilitate analysis and visualisation of time-resolved data from FLIM (Fluorescence Lifetime Imaging). FLIMfitt can connect to OMERO to load data directly from the OMERO server, or work with local files.
+---
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<a href="http://www.openmicroscopy.org/site/products/partner/flimfit" target="_blank" class="external-url"><img alt="../images/flimfitImperialLogo.jpg" class="align-left" src="images/flimfitImperialLogo.jpg" style="float: right;width: 25%;"/></a>
+
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#install">Installing FLIMfit</a><br />
+<a href="#using">Using FLIMfit with OMERO</a><br />
+<a href="#disk">Loading data from disk</a><br />
+<a href="#single">Fitting data - single image</a><br />
+<a href="#results">Viewing Results</a><br />
+<a href="#multiple">Fitting data - multiple images</a><br />
+<a href="#other">Other OMERO interactions</a><br />
+
+</p>
+</div><!-- End Page Navigation -->
+
+<div class="line-block">
+<div class="line"><br /></div>
+</div>
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
+
+<p>FLIMfit is a software tool that is designed to facilitate analysis and visualisation of time-resolved data from FLIM (Fluorescence Lifetime Imaging) measurements including time-correlated single photon counting (TCSPC) and wide-field time-gated imaging. FLIMfitt can connect to OMERO to load data directly from the OMERO server, or work with local files.</p>
```

Comment:
'FLIMfitt' typo in last sentence of this paragraph (sentence missing altogether from pdf)